control/pickupitems.txt

```diff
@@ -7,10 +7,6 @@
 #		 1 to pickup the item,
 #		 2 to take the item as fast as it can
 #
-#<item name> - name of item (not case sensitive)
-#
-#<flag> - 1 to pickup the item, 0 to leave it alone
-#
```

Comment:
Syncs control-br folder with main control folder
Removes duplicate explanation in pickupitems.txt and shop.txt